core/src/actors/inventory_manager/mod.rs

```diff
@@ -0,0 +1,10 @@
+//! # InventoryManager actor
+//!
+//! This module contains the InventoryManager actor which is in charge
+//! of managing inventory structures such as transactions, blocks or data requests.
```

Comment:
Hey hey hey I just realized that the description from #209 is not in line with the workflow we discussed recently. Let me edit it and I'll ping you back when it's right. Sorry for the hassle!

Reply:
Now it's OK. I just removed the paragraph about validations. It said that inventory items should be validated before sending them to `InventoryManager`, but we recently stated that `InventoryManager` should be the one triggering the validations so that other actors can abstract from that and just treat all inventory items in the same way.